onfido-java/src/main/java/com/onfido/api/ApiJson.java

```diff
@@ -1,5 +1,8 @@
 package com.onfido.api;
 
+import com.onfido.exceptions.OnfidoException;
+import com.squareup.moshi.*;
```

Comment:
What's the company policy (if any) on using wildcard imports?

Reply:
According to @ciaran16  there isn't one. I've gone ahead and updated them.